ReactNative/ReactTwitter/core/views/debug/push-notifications.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Text
+} from 'react-native'
+var Button = require('react-native-button')
+var NetworkCallsManager = require('../../service/network-calls-manager.js')
+var PushNotificationService = require('../../service/push-notifications-service.js')
+
+const PUSH_SERVER_URL = 'https://gcm-http.googleapis.com/gcm/send'
+const PUSH_DELAY_MS = 2000
+const PUSH_AUTHORIZATION_KEY = 'key=AIzaSyDiQB51T00e8s6JCkLYuvl0Ac05R0Dc4iY'
+
+var PushNotificationsView = React.createClass({
```

Comment:
This is more likely a Scene rather than a view. 

Can you update also the name of the file?